src/app/routes/enrolment/enrolment.component.html

```diff
@@ -4,8 +4,10 @@
       <div class="col-xl-12">
         <div class="d-flex flex-column flex-md-row justify-content-between align-items-center">
           <h4 class="color-white strong my-4">Issue Role</h4>
-          <button *ngIf="isExperimentalEnabled$ | async" mat-raised-button
-                  class="btn btn-primary btn-small mb-3 mb-lg-0" (click)="createVC()">Create VC
+          <button *ngIf="(isExperimental$ | async) && controlsRoles" mat-raised-button
```

Comment:
Sorry about this. I think my previous comment was pretty confusing. `controlsRoles` was the right name for the previous behaviour. But I think now that the behaviour has been corrected, `issuesRoles` is a better name.
```suggestion
          <button *ngIf="(isExperimental$ | async) && issuesRoles" mat-raised-button
```

Reply:
No problem :) Naming sometimes can be hard or tricky... I adjusted PR, thanks for checking!